pkg/controller/job/utils_test.go

```diff
@@ -24,26 +24,59 @@ import (
 )
 
 func TestIsJobFinished(t *testing.T) {
-	job := &batch.Job{
-		Status: batch.JobStatus{
-			Conditions: []batch.JobCondition{{
-				Type:   batch.JobComplete,
-				Status: v1.ConditionTrue,
-			}},
+	testCases := map[string]struct {
+		conditionType   batch.JobConditionType
+		conditionStatus v1.ConditionStatus
+		expect          bool
```

Comment:
I'm not sure what `expect` means? Would you name it `expectJobFinished`, and then return error `if expectJobFinished != IsJobFinished`

Reply:
OK,thanks~